### ISSUE TITLE
Improve keyboard accessibility of NetLogo Web

### ISSUE THREAD

**Issue by TheBizzle** [NetLogo/Galapagos#448]
@jzkelter tells me that

> VPAT ("Voluntary Product Accessibility Template") [is something that] many universities require (and probably public schools) before you can be a vendor for them. It includes this "All [functionality](https://www.w3.org/TR/WCAG20/#functiondef) of the content is operable through a [keyboard interface](https://www.w3.org/TR/WCAG20/#keybrd-interfacedef) without requiring specific timings for individual keystrokes..."

In this regard, we are deficient in a select few areas.

- [x] You can't tab to "forever" buttons (which just seems like a bug)
- [x] You can tab through most controls and interactable widgets, but not to the "Code" (etc.) panes
- [x] You should be able to escape text areas (e.g. the Code tab's editor) to resume tabbing through the page
  * My suggested solution: `Esc` jumps the focus up out of any editor
- [x] While not strictly necessary for this, it would nice to be able to quickly jump between the "Command Center", "Info", and "Code" panes
  * My suggested solution: In desktop NL, we use `Ctrl` or `Command` (i.e. `Ctrl+1`, `Ctrl+2`, `Ctrl+3`).  This _does_ pose a point of contention in the browser, since those are theoretically hotkeys used for navigating tabs.  But, honestly, I doubt that people use those shortcuts nearly as much as `Ctrl+Tab` and `Ctrl+Shift+Tab`.  We're probably fine overriding those shortcuts.
- [x] The current widget tabbing order can seem very random, because it's probably based off of whatever the order of the widgets is in the `.nlogox` file.  It might make sense to assign tab order based on position, left to right, top to bottom.  Maybe. 🤷‍♂️
